skimage/transform/radon_transform.py

```diff
@@ -62,6 +63,8 @@ def radon(image, theta=None, circle=True):
     if theta is None:
         theta = np.arange(180)
 
+    image = convert_to_float(image, preserve_range=True)
```

Comment:
This needs a new parameter, and documentation. I think our default is to have `preserver_range=False`.

You can consider leaving this for a different PR and creating an issue about it.

Reply:
Should I create an Issue about something that is not yes in `skimage` code?

Reply:
I though about leaving `preserve_range` to its default value, but this would potentially break users code, but well, I can open an issue about the fact that the arrays are implicitly casted to `double` by `_warp_cy` in `radon` ;-)